lib/rubocop/cop/style/redundant_string_escape.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for redundant escapes in Strings.
+      #
+      # @example
```

Comment:
Probably you should add a couple of examples with heredocs as well, so it's clear to everyone those are considered.